cypress/integration/seededFlows/profileFlows/followUser.spec.js

```diff
@@ -14,22 +14,26 @@ describe('Follow user from profile page', () => {
 
     cy.findByRole('button', {
       name: 'Follow user: Article Editor v1 User',
-    }).click();
+    }).as('followButton');
+    cy.get('@followButton').click();
     cy.wait('@followsRequest');
-    cy.findByRole('button', { name: 'Unfollow user: Article Editor v1 User' });
+
+    cy.get('@followButton').should('have.text', 'Following');
+    cy.get('@followButton').should('have.attr', 'aria-pressed', 'true');
 
     // Check that the update persists after reload
     cy.visitAndWaitForUserSideEffects('/article_editor_v1_user');
 
-    cy.findByRole('button', {
-      name: 'Unfollow user: Article Editor v1 User',
-    }).click();
```

Comment:
as above, this step was originally in to check that the correct state is initialized on reload, so we should reintroduce this line and check the aria-pressed state is still 'true'